docs/hub/academia-hub.md

```diff
@@ -0,0 +1,37 @@
+# Academia Hub
+
+<Tip>
+<a href="https://huggingface.co/contact/sales?from=academia" target="_blank">Get in touch</a> from your university-affiliated email address to initiate the subscription process.
+</Tip>
+
+Academia Hub provides every student, reseacher, of faculty member having a verified and currently active email address from your academic institution access to our Hub’s PRO features. Explore advanced tools, enhanced collaboration, and exclusive resources to accelerate your learning and research.
+
+<a href="https://huggingface.co/contact/sales?from=academia" class="flex justify-center">
+    <img class="block" src="https://huggingface.co/datasets/julien-c/documentation-images/resolve/main/AcademiaHub.png" />
+</a>
+
+#### How does it work
+
+Any affiliated user will need to add their `@your-university-name.edu` email address to their HF account.
+
+Academia Hub is designed for:
+- **Students:** Unlock powerful features to learn about AI and Machine learning in the most efficient way.
+- **Researchers:** Collaborate with peers using the standard AI ecosystem of tools.
+- **Faculty members:** Enhance your classes' projects with PRO capabilities.
+
+Key Features are:
+
+- **Full PRO Access:** Enjoy all premium functionalities on the Hub.
+- **Seamless Onboarding:** Quick and secure sign-up with your academic email.
+- **Enhanced Collaboration:** Connect with peers and mentors.
+- **Exclusive Resources:** Access datasets, models, and projects tailored for academia.
+
+#### Eligibility
+
+- Must possess a valid university or college email address.
+- Open to all students regardless of discipline or level of study.
+- Pricing: Academia Hub is priced based on volume of usage and number of active users. 
```

Comment:
Can they self-subscribe or is it in a per-case basis?

Reply:
i think it's through a contact with our team only